src/styles/blocks/_catalog.scss

```diff
@@ -0,0 +1,29 @@
+.catalog {
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(4, auto);
+  gap: 48px;
+  margin: 0 auto;
+
+  @media(min-width: 768px) and (max-width: 1023px) {
+    grid-template-columns: repeat(3, auto);
+  }
+
+  @media(min-width: 488px) and (max-width: 767px) {
+    grid-template-columns: repeat(2, auto);
+  }
+
+  @media(max-width: 487px) {
+    grid-template-columns: repeat(1, auto);
+  }
```

Comment:
Rewrite the media queries to have only one condition each.
Choose the direction (1 -> 4 or 4 -> 1) and follow it.

Reply:
Fixed.